listenbrainz/webserver/templates/base.html

```diff
@@ -4,60 +4,139 @@
+          </p>
+          <ul class="list-unstyled">
+            <li class="color-a">
+              <span class="color-gray">IRC: </span> <a href="https://kiwiirc.com/nextclient/irc.libera.chat/?#metabrainz" target="_blank" rel="noopener noreferrer" > #metabrainz</a>
+            </li>
+            <li class="color-a" >
+              <span class="color-gray">Email: </span> <a href="mailto:support@metabrainz.org">support@metabrainz.org </a>
+            </li>
+          </ul>
+        </div>
+        <br/>
+        <div class="col-sm-12 col-md-3 section-md-t3">
+          <h3 class="w-title-a text-brand">Useful Links</h3>
+          <ul class="list-unstyled">
+            <li class="item-list-a">
+              <img src="{{ url_for('static', filename='img/icons/angle_double_right_icon.svg') }}" width="24" height="24" alt="image"/> <a href="https://metabrainz.org/donate" target="_blank" rel="noopener noreferrer" >Donate</a>
+            </li>
+            <li class="item-list-a">
+              <img src="{{ url_for('static', filename='img/icons/angle_double_right_icon.svg') }}" width="24" height="24" alt="image"/> <a href="https://wiki.musicbrainz.org/Main_Page" target="_blank" rel="noopener noreferrer" >Wiki</a>
+            </li>
+            <li class="item-list-a">
+              <img src="{{ url_for('static', filename='img/icons/angle_double_right_icon.svg') }}" width="24" height="24" alt="image"/><a href="https://community.metabrainz.org/" target="_blank" rel="noopener noreferrer" >Community</a>
+            </li>
+            <li class="item-list-a">
+              <img src="{{ url_for('static', filename='img/icons/angle_double_right_icon.svg') }}" width="24" height="24" alt="image"/><a href="https://blog.metabrainz.org/" target="_blank" rel="noopener noreferrer" >Blog</a>
+            </li>
+            <li class="item-list-a">
+              <img src="{{ url_for('static', filename='img/icons/angle_double_right_icon.svg') }}" width="24" height="24" alt="image"/> <a href="https://www.redbubble.com/people/metabrainz/shop" target="_blank" rel="noopener noreferrer" >Shop</a>
+            </li>
+            <li class="item-list-a">
+              <img src="{{ url_for('static', filename='img/icons/angle_double_right_icon.svg') }}" width="24" height="24" alt="image"/> <a href="https://metabrainz.org/" target="_blank" rel="noopener noreferrer" >MetaBrainz</a>
+            </li>
+
+          </ul>
+        </div>
+        <div class="col-sm-12 col-md-3 section-md-t3">
+          <h3 class="w-title-a text-brand">Fellow Projects</h3>
+          <ul class="list-unstyled">
+            <li class="item-list-a">
+              <img src="{{ url_for('static', filename='img/icons/angle_double_right_icon.svg') }}" width="24" height="24" alt="image"/>  <img src="{{ url_for('static', filename='img/meb-icons/MusicBrainz.svg') }}" width="24" height="24" alt="image"/> <a href="https://musicbrainz.org/" target="_blank" rel="noopener noreferrer" >MusicBrainz</a>
```

Comment:
In complement to what I said above about `alt` descriptions, here for the project logos images we want to change that to `alt="MusicBrainz logo` etc.

Reply:
Done

Reply:
Done